tests/scripts/thread-cert/Cert_5_3_07_DuplicateAddress.py

```diff
@@ -117,7 +117,13 @@ def test(self):
         self.nodes[ED2].add_ipaddr('2001::1')
         time.sleep(3)
 
-        self.nodes[ED3].ping('2001::1')
+
+        # Final ping doesn't need a response;
+        # used to trigger an ADDR_ERR.ntf
+        try:
+            self.nodes[ED3].ping('2001::1')
+        except pexpect.TIMEOUT:
```

Comment:
Here is the debug output for Cert_5_3_07 after setting up the duplicate address situation.  This trace is from when the ping is sent to the initial timeout.  It looks like after sending the ping, the host gets back "no route", no response, and then the timeout.

```
### ===> Node 6 issues CLI command to HOST driver
6: ping 2001::1

### ===> Some asynchronous inbound traffic before command is issued at Spinel level
spinel-cli > RX Pay: (99) ['80', '06', '73', '5C', '00', '60', '00', '00', '00', '00', '34', '11', 'FF', 'FE', '80', '00', '00', '00', '00', '00', '00', 'F0', 'D9', '2A', '82', 'C8', 'D8', 'FE', '43', 'FF', '02', '00', '00', '00', '00', '00', '00', '00', '00', '00', '00', '00', '00', '00', '01', '4D', '4C', '4D', '4C', '00', '34', 'F2', '78', '00', '15', '10', '00', '00', '00', '00', '00', '00', '00', '01', '16', '47', 'FC', '46', '5B', '58', 'F7', '15', 'D5', '7C', '36', '52', 'EA', 'CD', '23', 'CE', '96', 'DD', '3D', '24', 'BC', 'DC', '97', 'DF', 'E6', '56', '03', 'C9', '72', '29', 'A5', '11', 'B9', 'F5', 'F3'] 
RX Pay: (87) ['80', '06', '73', '50', '00', '60', '00', '00', '00', '00', '28', '11', 'FF', 'FE', '80', '00', '00', '00', '00', '00', '00', 'F0', 'D9', '2A', '82', 'C8', 'D8', 'FE', '43', 'FF', '02', '00', '00', '00', '00', '00', '00', '00', '00', '00', '00', '00', '00', '00', '01', '4D', '4C', '4D', '4C', '00', '28', '60', '02', '00', '15', '11', '00', '00', '00', '00', '00', '00', '00', '01', '2B', 'D8', '6E', 'B1', '65', '1F', '36', '1E', '11', '14', '98', '03', '27', 'C4', '70', 'BB', '9F', '94', '62', '1F', '78', '23', 'D0'] 
ping 2001::1
TX Pay: (3) ['81', '02', '61'] 
RX Pay: (5) ['80', '06', '5A', '2F', '63'] 
PROP_VALUE_IS: THREAD_STABLE_NETWORK_DATA = None
RX Pay: (105) ['80', '06', '73', '62', '00', '60', '00', '00', '00', '00', '3A', '11', 'FF', 'FE', '80', '00', '00', '00', '00', '00', '00', 'F0', 'D9', '2A', '82', 'C8', 'D8', 'FE', '43', 'FE', '80', '00', '00', '00', '00', '00', '00', 'A4', '1A', 'FE', '0F', 'B1', '10', 'F8', '96', '4D', '4C', '4D', '4C', '00', '3A', 'F9', 'B6', '00', '15', '12', '00', '00', '00', '00', '00', '00', '00', '01', 'A7', '5D', 'A1', '19', '11', '80', '4C', '38', '0D', 'D8', '00', '95', 'EA', '04', 'F5', '7C', '9A', '83', '1A', '43', '90', '59', '79', 'BF', 'EE', 'EB', '3D', '6F', 'DE', '62', 'A3', 'F1', '8D', '5B', 'E3', '41', '23', 'E1', '33', '92', '16'] 
RX Pay: (21) ['81', '06', '61', 'FD', 'DE', 'AD', '00', 'BE', 'EF', '00', '00', '23', '15', 'C1', '82', '07', '32', '2E', '9B', '2C', '1F'] 

### ===> Actual send of ping6 2001::1 over spinel to NCP
TX Pay: (55) ['81', '03', '72', '30', '00', '60', '00', '00', '00', '00', '08', '3A', '40', 'FD', 'DE', 'AD', '00', 'BE', 'EF', '00', '00', '23', '15', 'C1', '82', '07', '32', '2E', '9B', '20', '01', '00', '00', '00', '00', '00', '00', '00', '00', '00', '00', '00', '00', '00', '01', '80', '00', 'DB', '86', '00', '00', '00', '00', '00', '00'] 

### ===> Spinel returns a Thread stack error
RX Pay: (8) ['81', '06', '00', '84', '80', '01', '56', '3F'] 
PROP_VALUE_IS: LAST_STATUS = 16388
kThreadError_NoRoute
E1: thread stop
```